src/app/(pages)/fhirServers/page.tsx

```diff
@@ -461,64 +467,78 @@ const FhirServers: React.FC = () => {
             </tr>
           </thead>
           <tbody>
-            {fhirServers.map((fhirServer) => (
-              <tr
-                key={fhirServer.id}
-                className={classNames(styles.tableRowHover)}
-              >
-                <td>{fhirServer.name}</td>
-                <td>{fhirServer.hostname}</td>
-                <td>
-                  {fhirServer.authType ||
-                    (fhirServer.headers?.Authorization ? "basic" : "none")}
-                </td>
-                <td width={480}>
-                  <div className="grid-container grid-row padding-0 display-flex flex-align-center">
-                    {fhirServer.lastConnectionSuccessful ? (
-                      <>
-                        <Icon.Check
-                          size={3}
-                          className="usa-icon margin-right-05"
-                          aria-label="Connected"
-                          color="green"
-                        />
-                        Connected
-                      </>
-                    ) : (
-                      <>
-                        <Icon.Close
-                          size={3}
-                          className="usa-icon margin-right-05"
-                          aria-label="Not connected"
-                          color="red"
-                        />
-                        Not connected
-                      </>
-                    )}
-                    <span className={styles.lastChecked}>
-                      (last checked:{" "}
-                      {fhirServer.lastConnectionAttempt
-                        ? new Date(
-                            fhirServer.lastConnectionAttempt,
-                          ).toLocaleString()
-                        : "unknown"}
-                      )
-                    </span>
-                    <button
-                      className={classNames(
-                        styles.editButton,
-                        "usa-button usa-button--unstyled",
+            {fhirServers
+              .slice()
+              .sort((a, b) =>
+                b.defaultServer === true
+                  ? 1
+                  : a.defaultServer === true
+                    ? -1
+                    : 0,
+              )
+              .map((fhirServer) => (
+                <tr
+                  key={fhirServer.id}
+                  className={classNames(styles.tableRowHover)}
+                >
+                  <td>
+                    {fhirServer.name}{" "}
+                    {fhirServer.defaultServer ? (
+                      <Tag className="margin-left-2">DEFAULT</Tag>
```

Comment:
We might need to tweak the way the tag component wraps here since it seems to not align correctly when the name gets too long. Worth tagging design in case we haven't already?

<img width="1188" alt="Screenshot 2025-05-02 at 10 14 11 AM" src="https://github.com/user-attachments/assets/2de9324c-9bc5-4da2-a493-d202be5a54ba" />